test/support/iterator_utils.h

```diff
@@ -257,6 +257,12 @@ using is_base_of_iterator_category = ::std::is_base_of<Tag, typename iterator_tr
 template <typename Tag, typename Iter>
 inline constexpr bool is_base_of_iterator_category_v = is_base_of_iterator_category<Tag, Iter>::value;
 
+template <typename Iter>
+using is_base_of_iterator_random_access = is_base_of_iterator_category<::std::random_access_iterator_tag, Iter>;
+
+template <typename Iter>
+inline constexpr bool is_base_of_iterator_random_access_v = is_base_of_iterator_random_access<Iter>::value;
```

Comment:
Technically it's correct.
But in tests there are a lot of places when we should use this define too,
But we still use `::std::enable_if_t<is_base_of_iterator_category<::std::random_access_iterator_tag, OutputIterator>::value>` and so far and so on,
So my proposal - to use `is_base_of_iterator_category_v` in this test too.

Reply:
I've removed this in favor of just using the existing infrastructure.

Reply:
OK